### ISSUE TITLE
Use 3rd-party authentication

### ISSUE THREAD

**Issue by subelsky** [ignitebaltimore/ignitespeak#7]
This would replace our current simple HTTP basic auth system.

Ignite admins should be able to authenticate themselves via Twitter, Facebook, or Google Apps. Hopefully there's a gem already that does all 3 of those. Or it could use a 3rd-party persona management SaaS product. Definitely do not want to roll our own authentication code.

We would also need a `User` model, accessible via `rails_admin` that lets you add new users easily.

We would also need a rake task that can be run by the account owner from the command line, so at least the first user could be added:

``` ruby
heroku run rake users:add mike@ignitebaltimore.com
```

**Comment by wistcax** [comment]
@subelsky I have added Twitter and Google authentication and removed the basic https authentication. The admin would need to create his/her own email and password in the admin database. File db/seed.rb shows how to do this. Did not do Facebook because I don't have it.
Is it what you wanted? Thx.

**Comment by subelsky** [comment]
Yes! It's very close. I made a bunch of comments (I think I reviewed some commits out of order so apologies if I said to do something you've already done in a subsequent commit!).

Main thing now is to add testing

**Comment by wistcax** [comment]
sorry, did not know you replied. Will sort it out - not sure this week though.
